contentctl/objects/detection_tags.py

```diff
@@ -49,6 +49,23 @@ class DetectionTags(BaseModel):
     @property
     def risk_score(self) -> int:
         return round((self.confidence * self.impact)/100)
+    
+    @computed_field
+    @property
+    def severity(self)->RiskSeverity:
+        if 0 <= self.risk_score <= 20:
```

Comment:
The intent here is that these values have been taken from 
https://docs.splunk.com/Documentation/ES/7.3.2/RBA/Analyzerisk#Score_ranges_for_risk

Reply:
LGTM
For reference, severity here is actually a notable thing that determines urgency:
https://docs.splunk.com/Documentation/ES/7.3.2/User/Howurgencyisassigned